### ISSUE TITLE
Template Macros don't play properly

### ISSUE THREAD

**Issue by MrVauxs** [fantasycalendar/FoundryVTT-Sequencer#80]
**Describe the bug**
When attempting to play the below macro through Automated Animations + Advanced Macros, a "Temporary effect" is spawned that never ends, and the following order is posted to console.

```js
console.log(args[1].templateData)
const template = args[1].templateData;
new Sequence()
    .effect()
        .file('jb2a.energy_strands.range.multiple.dark_purple.01')
        .fadeIn(500)
        .fadeOut(500)
        .atLocation(template)
        .stretchTo(template)
        .scale({ x: 1.4 })
        .spriteOffset({ x: -150 })
    .play()
```

![image](https://user-images.githubusercontent.com/32039708/172442712-35625ba1-548a-499f-adf1-b38f2d018dc7.png)

![image](https://user-images.githubusercontent.com/32039708/172443988-91173699-daa3-4914-ac16-29eb2e32c9c3.png)

**Desktop (please complete the following information):**
 - Windows 10
 - Foundry Electron app

**Additional context**
Foundry 9.269
Pathfinder 2nd Edition 3.10.4.11747
Sequencer 2.1.0
Automated Animations 3.2.1
Advanced Macros 1.14.2
JB2A_patreon 0.4.4

**Comment by MrVauxs** [comment]
Additional console screenshots of `args[1]` from Automated Animations in PF2e system
![image](https://user-images.githubusercontent.com/32039708/172443734-a11599bf-eaec-4278-8f9f-e12699f54a05.png)
![image](https://user-images.githubusercontent.com/32039708/172443743-fea84dcc-b3a7-4d67-ac6b-b0565baebccc.png)
![image](https://user-images.githubusercontent.com/32039708/172443753-f8274055-61b6-4c00-896d-fd1be753f348.png)

**Comment by Haxxer** [comment]
Fixed in 2.1.1
